spynnaker/pyNN/models/utility_models/synapse_expander/synapse_expander.py

```diff
@@ -102,6 +82,36 @@ def synapse_expander(
                 "The synapse expander failed to complete")
 
 
+def _plan_expansion(app_graph, placements, synapse_expander_bin,
+                    delay_expander_bin, progress):
+    """ Works out exactly where we are going to be running expanders,\
+        and exactly which expander to run in each of those places.
+    """
+    expander_cores = ExecutableTargets()
+    gen_on_machine_vertices = list()
```

Comment:
You've renamed these to expanded_pop_vertices outside of this function, so they should probably be called that here too.